src/porepy/applications/md_grids/mdg_library.py

```diff
@@ -9,6 +9,8 @@
 from typing import Literal
 
 import porepy as pp
+from porepy.fracs.fracture_network_2d import FractureNetwork2d
```

Comment:
In this case is absolutely necessary (regardless of the issue mentioned by @vlipovac), since `pp.FractureNetwork2d` and `pp.FractureNetwork3d` were removed from the top `__init__` file.